build_calendar.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/python
+import json
+import sys
+from datetime import datetime
+import os
+import time
+import icalendar
+import requests
+import settings
+
+from calgen.mixins import GlobalHolidays
+from calgen.models.Calendar import CalendarTypes
+from calgen.models.Calendarific import Calendarific
+
+class CountryNotSupportedException(Exception):
+    def __init__(self, country):
+        self.country = country
+
+
+def query_calendarific(api_key, country, year, calendar_type):
+    if country not in settings.CALENDAR_LOCALES:
```

Comment:
I might be confused by something here, but how can this be called with a country that isn't in `CALENDAR_LOCALES` in the first place? It looks like `build_calendars` is called with `CALENDAR_LOCALES` as the initial list.